docs/getting_started/faq.rst

```diff
@@ -0,0 +1,131 @@
+
+How much faster can I go with Modin compared to pandas?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Modin is designed to scale with the amount of hardware available.
+Even in a traditionally serial task like ``read_csv``, we see large gains by efficiently 
+distributing the work across your entire machine. Because it is so light-weight, 
+Modin provides speed-ups of up to 4x on a laptop with 4 physical cores. This speedup scales
+efficiently to larger machines with more cores. We have several published papers_ that
+include performance results and comparisons against pandas.
+
+How much more data would I be able to process with Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Often data scientists have to use different tools for operating on datasets of different sizes. 
+This is not only because processing large dataframes is slow, but also pandas does not support working 
+with dataframes that don't fit into the available memory. As a result, pandas workflows that work well 
+for prototyping on a few MBs of data do not scale to tens or hundreds of GBs (depending on the size 
+of your machine). Modin supports operating on data that does not fit in memory, so that you can comfortably 
+work with hundreds of GBs without worrying about substantial slowdown or memory errors. For more information, 
+see :doc:`out-of-memory support <getting_started/out_of_core.rst>` for Modin.
+
+How does Modin work under the hood?
+""""""""""""""""""""""""""""""""""""
+
+Modin is logically separated into different layers that represent the hierarchy of a 
+typical Database Management System. User queries which perform data transformation, 
+data ingress or data egress pass through the Modin query compiler which translates 
+queries from the top-level pandas API Layer that users interact with to the Modin Core DataFrame_ layer. 
+The Modin Core DataFrame is our efficient DataFrame implementation that utilizes a partitioning schema 
+which allows for distributing tasks and queries. From here, the Modin DataFrame works with engines like
+Ray or Dask to execute computation, and then return the results to the user.
+
+For more details, take a look at our system :doc:`architecture </developer/architecture>`. 
+
+If I’m only using my laptop, can I still get the benefits of Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Absolutely! Unlike other parallel DataFrame systems, Modin is an extremely 
+light-weight, robust DataFrame. Because it is so light-weight, Modin provides 
+speed-ups of up to 4x on a laptop with 4 physical cores 
+and allows you to work on data that doesn't fit in your laptop's RAM.
+
+How do I use Jupyter or Colab notebooks with Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+You can take a look at this Google Colab installation guide_ and
+this notebook tutorial_. Once Modin is installed, simply replace your pandas
```

Comment:
Maybe we should put a link to https://github.com/modin-project/modin/tree/master/examples/tutorial?

Reply:
This is a much better page to link to, thanks!